wandb/jupyter.py

```diff
@@ -359,8 +359,9 @@ def save_history(self):
                 },
             )
             state_path = os.path.join("code", "_session_history.ipynb")
-            wandb.run.config["_wandb"]["session_history"] = state_path
-            wandb.run.config.persist()
+            wandb.run._config_callback(
+                val=state_path, key=("_wandb", "session_history")
+            )
```

Comment:
nice catch, can you instead put a private method on wandb.run (or config)
I think we used to have one called `run._set_config_wandb` 

That will be cleaner than referring to the callback here.

Reply:
Can do!